Applications/Spire/Include/Spire/Ui/StaticDropDownMenu.hpp

```diff
@@ -0,0 +1,106 @@
+#ifndef SPIRE_STATIC_DROP_DOWN_MENU_HPP
+#define SPIRE_STATIC_DROP_DOWN_MENU_HPP
+#include <QTimer>
+#include <QWidget>
+#include "Spire/Spire/Spire.hpp"
+#include "Spire/Ui/CustomQtVariants.hpp"
+#include "Spire/Ui/DropDownList.hpp"
+
+namespace Spire {
+
+  //! Represents a DropDownMenu that isn't modified by user interactions.
+  class StaticDropDownMenu : public QWidget {
+    public:
+
+      //! Signals that an item was selected.
+      /*!
+        \param index The index of the selected item.
+      */
+      using IndexSelectedSignal = Signal<void (int index)>;
+
+      //! Signals that an item was selected.
+      /*!
+        \param value The value of the selected item.
+      */
+      using ValueSelectedSignal = Signal<void (const QVariant& value)>;
+
+      //! Constructs a StaticDropDownMenu.
+      /*!
+        \param items The items to display.
+        \param parent The parent widget.
+      */
+      explicit StaticDropDownMenu(const std::vector<QVariant>& items,
+        QWidget* parent = nullptr);
+
+      virtual ~StaticDropDownMenu() = default;
+
+      //! Constructs a StaticDropDownMenu with a fixed input label.
+      /*!
+        \param items The items to display.
+        \param display_text The fixed input label text to display.
+        \param parent The parent widget.
+      */
+      StaticDropDownMenu(const std::vector<QVariant>& items,
+        const QString& display_text, QWidget* parent = nullptr);
+
+      //! Returns the number of items in the list.
+      virtual int item_count() const;
```

Comment:
Not sure if we code natural numbers with signed integers in public methods, please confirm.